### PR TITLE
Updates to kernel-rt package (5.15.55.1)

### PR DESCRIPTION
**Pull request by xfoukas**
* Move to kernel-rt 5.15.55.1

* Changes to config file to harmonize with non-rt kernel

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

This PR moves the kernel-rt package to 5.15.55.1 and harmonizes the config file to the non-rt kernel.

###### Change Log  

- Change Move package kernel-rt to 5.15.55.1.

###### Does this affect the toolchain?  

NO

###### Test Methodology

- Local build

### CODE REVIEW

**Review by Camelron — APPROVED** [review]
looks good to me

**Review by snambakam — APPROVED** [review]

**Review by mfrw — CHANGES_REQUESTED** [review]
~Aplogies for blocking this PR:
It seems our BuddyBuild is not happy: [23061](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=230613&view=logs&j=75cba72d-09cb-503a-3927-4ab61ac4a179&t=3302d08b-c63b-5f49-5695-b9171bdc6cac&l=12)
Can we please double check once :)~

EDIT: I take my words back.

**Review by mfrw — APPROVED** [review]

File: SPECS/kernel-rt/kernel-rt.spec [hunk 1]
```diff
@@ -28,6 +28,7 @@ Source3:        cbl-mariner-ca-20211013.pem
 # Also, remember to bump the global rt_version macro above ^
 Patch0:         patch-%{version_upstream}-%{rt_version}.patch
 Patch1:         realtime-with-memcg.patch
+Patch2:         intel-ice.patch
```
Comment (vinceaperri) [inline SPECS/kernel-rt/kernel-rt.spec:31]: Need to also add `%patch2 -p1` below to apply the patch.

**Review by Camelron — CHANGES_REQUESTED** [review]
Let's add the %patch2 -p1 call and verify that things are still building

**Review by Camelron — APPROVED** [review]

**Review by vinceaperri — APPROVED** [review]

**Review by christopherco — APPROVED** [review]

**Pull request merged**